keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -89,6 +91,12 @@ class SqlResourceRepository(
       .onDuplicateKeyUpdate()
       .set(updatePairs)
       .execute()
+    jooq.insertInto(RESOURCE_LAST_CHECKED)
```

Comment:
Do we need to specify that this should happen as a transaction (or are we specifying that in some what that I don't understand)?

Reply:
The transaction boundary is in `ResourcePersister`. I want to add `@Transactional(REQUIRED)` or `@Transactional(SUPPORTS)` to the relevant methods in the repository classes, but only adding it on the `SQL*` ones messed with the `spring-kotlin` Gradle plugin and adding it on the interface seems meaningless when in-memory, etc. probably couldn't be guaranteed to abide by it.

Reply:
Raised #354 to track this